frontend/playwright.config.ts

```diff
@@ -76,7 +103,7 @@ const config: PlaywrightTestConfig = {
   },
 
   /* Opt out of parallel tests. */
-  workers: 1,
+  // workers: 1,
```

Comment:
Ah I thought running parallel tests in CI would slow down actual test time like it used to be, but that doesn't seem to be the case anymore. Thanks for updating this!

```suggestion
  /**
   * By default Github Action's hosted runner has 2 CPUs, so Playwright will
   * spin up NUM_CPU / 2 workers, which is 1 worker. But locally it will run
   * tests with more workers
   * https://github.com/microsoft/playwright/issues/19408#issuecomment-1347341819
   */
  // workers: 1,
```